tests/m2m_through/models.py

```diff
@@ -83,6 +83,20 @@ class Friendship(models.Model):
     date_friended = models.DateTimeField()
 
 
+class PersonSelfRefM2MSymmetrical(models.Model):
+    name = models.CharField(max_length=5)
+    friends = models.ManyToManyField('self', through="SymmetricalFriendship", symmetrical=True)
+
+    def __str__(self):
+        return self.name
+
+
+class SymmetricalFriendship(models.Model):
+    first = models.ForeignKey(PersonSelfRefM2MSymmetrical, models.CASCADE, related_name="+")
+    second = models.ForeignKey(PersonSelfRefM2MSymmetrical, models.CASCADE)
```

Comment:
I know this is a test so it doesn't matter as much, but I think it would make more sense to keep the `related_name="+"` on `second` so that if you are trying to reference `symmetricalfriendship_set`, it will fetch the one where the caller `PersonalSelfRefM2MSymmetrical` object is `first`.  Strictly speaking, there isn't a defined directionality, but `first` to `second` seems like the more intuitive one (you have this correct in the [added documentation for this PR](https://github.com/django/django/pull/11298/files#diff-7e6909c7694e8a3ae170c2a22f9b6e0cR1577)).

```python
tony = PersonSelfRefM2MSymmetrical.objects.create(name="Tony")
chris = PersonSelfRefM2MSymmetrical.objects.create(name="Chris")
tony.friends.add(chris)

tony.symmetricalfriendship_set.first() == SymmetricalFriendship.objects.get(first=tony)  # False
tony.symmetricalfriendship_set.first() == SymmetricalFriendship.objects.get(second=tony)  # True
```

Might also be worth adding a test for this.

Reply:
This made this part a bit clearer in my head, thank you. 
I added a test and updated the documentation too.